.github/nightly-downloads.template.html

```
@@ -0,0 +1,264 @@
+      <header class="hero">
+        <picture>
+          <source media="(prefers-color-scheme: dark)" srcset="https://amaru.global/_astro/logo-dark.De0RyNtz.svg" />
+          <source media="(prefers-color-scheme: light)" srcset="https://amaru.global/_astro/logo-light.C5lipD4m.svg" />
+          <img alt="Amaru" src="https://amaru.global/_astro/logo-dark.De0RyNtz.svg" />
+        </picture>
+      </header>
+
+__LATEST_RELEASE_SECTION__
+
+      <section class="section">
+        <h2>Latest nightly build<h2>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </h2></h2> ], start tag match failed [ <h2> ] on line 186.

(tag-pair)